client/my-sites/patterns/components/pattern-preview/index.tsx

```diff
@@ -139,12 +140,13 @@ function PatternPreviewFragment( {
 
 export function PatternPreview( props: PatternPreviewProps ) {
 	const { isResizable, pattern } = props;
+	const isMobile = useMobileBreakpoint();
 
 	if ( ! pattern ) {
 		return null;
 	}
 
-	if ( ! isResizable ) {
+	if ( ! isResizable || isMobile ) {
```

Comment:
Great catch! It definitely doesn't make sense to allow for resize on mobile! The fix also ensures the pattern iframe renders properly without running beyond the screen width.

|Before|After|
|-------|-----|
|<img width="393" alt="Screen Shot 2024-03-21 at 11 22 23 AM" src="https://github.com/Automattic/wp-calypso/assets/104910361/56069d03-07a1-4d85-b721-6a5d37efb771">|<img width="391" alt="Screen Shot 2024-03-21 at 11 22 29 AM" src="https://github.com/Automattic/wp-calypso/assets/104910361/90b1c141-c6aa-485c-83a0-727343202fdc">|